src/main/java/com/google/identitytoolkit/GitkitClient.java

```diff
@@ -176,6 +176,79 @@ public GitkitUser validateTokenInRequest(HttpServletRequest request)
     }
     return null;
   }
+  
+  /**
+   * Verifies the user entered password at Gitkit server.
+   *
+   * @param email The email of the user
+   * @param password The password inputed by the user
+   * @param pendingIdToken The GITKit token for the non-trusted IDP, which is to be confirmed by the user
+   * @param captchaChallenge The captcha challenge
```

Comment:
Apologies for not saying the background clearly. Gitkit has upgraded to use reCaptcha v2, which wraps all parameters in the captchaResponse param. Therefore the method could be: verifyPassword(String email, String password, String pendingIdToken, String captchaResponse).